index.html

```diff
@@ -3244,6 +3294,108 @@ <h3 id="applying">
+          an application</dfn> are given by the following algorithm. The
+          algorithm takes an <a>application object</a> <var>application</var>.
+          This algorithm will return a string or <code>undefined</code>.
+        </p>
+        <ol>
+          <li>Let <var>value</var> be the result of calling the
+          <a>[[\GetOwnProperty]]</a> internal method of <var>application</var>
+          passing "<code>min_version</code>" as the argument.
+          </li>
+          <li>If <a>Type</a>(<var>value</var>) is not "string":
+            <ol>
+              <li>If <a>Type</a>(<var>value</var>) is not "undefined",
+              optionally <a>issue a developer warning</a> that the type is not
+              supported.
+              </li>
+              <li>Return <code>undefined</code>.
+              </li>
+            </ol>
+          </li>
+          <li>Otherwise, <a>Trim</a>(<var>value</var>) and return the result.
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          <code>fingerprints</code> member
+        </h3>
+        <p>
+          The <dfn data-lt="application-fingerprints">fingerprints</dfn> member
+          of an application object represents a set of cryptographic
+          fingerprints used for verifying the application. It is given as an
+          array of <a data-lt="fingerprint object">fingerprint objects</a>.
+        </p>
+        <p>
+          A <dfn>fingerprint object</dfn> has two properties: <code>type</code>
+          and <code>value</code>. Each of these are strings, but their
+          semantics are platform-defined.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>fingerprints</code> member of
+          an application</dfn> are given by the following algorithm. The
+          algorithm takes an <a>application object</a> <var>application</var>.
+          This algorithm will return a list of <a data-lt=
+          "fingerprint object">fingerprint objects</a>, possibly empty.
+        </p>
+        <ol>
+          <li>Let <var>fingerprints</var> be an empty list.
+          </li>
+          <li>Let <var>unprocessed fingerprints</var> be the result of calling
+          the <a>[[\GetOwnProperty]]</a> internal method of
+          <var>application</var> passing "<code>fingerprints</code>" as the
+          argument.
+          </li>
+          <li>If <var>unprocessed fingerprints</var> is an array, then for each
+          <var>potential fingerprint</var> in <var>unprocessed
+          fingerprints</var>:
+            <ol>
+              <li>Let <var>type</var> be the result of calling the
+              <a>[[\GetOwnProperty]]</a> internal method of <var>potential
+              fingerprint</var> passing "<code>type</code>" as an argument.
+              </li>
+              <li>If Type(<var>type</var>) is not "string", then issue a
+              developer warning that the type is unsupported, and move on to
```

Comment:
"issue a developer warning" should be linked.

I noticed that the other uses of this include the word "optionally" (see id and min_version). However, the definition of "issue a developer warning" clearly says that the UA can decide to do nothing, so the word "optionally" is redundant here. Perhaps remove it from the other usages?